.github/workflows/test_and_deploy.yml

```diff
@@ -52,12 +52,13 @@ jobs:
             python-version: "3.10"
 
     steps:
-      # Cache the tensorflow model so we don't have to remake it every time
-      - name: Cache tensorflow model
+      - name: Cache atlases
```

Comment:
Should this be changed to something more descriptive? I don't think we're actually using any atlases in the `cellfinder` tests so it would be more appropriate to call the step `Cache models` or even `Cache .brainglobe directory`

Reply:
Good idea, updated.